build.gradle

```diff
@@ -17,6 +17,9 @@ allprojects {
         maven {
             url 'http://oss.sonatype.org/content/repositories/snapshots'
         }
+        maven {
+            url 'https://jitpack.io'
```

Comment:
please TODO comment that this needs to be removed when real support is released.